tests/wast/wasmer/atomic_load.wast

```diff
@@ -0,0 +1,9 @@
+(module
+    (memory 1)
+    (func (export "atomic_load")
+        i32.const 0xffff_fff0
+        i32.atomic.load offset=16
+        drop
+    )
+)
+(assert_trap (invoke "atomic_load") "out of bound")

```

Comment:
I don't think the WAST tests can actuallly test concurrency, right?

Would be nice to have a custom test that actually runs multiple threads and tests notify/wait behaviour.

Reply:
Yeah, WAST will not really test concurency. We need a new kind of test suite for this, with probably some new infrastructure (so I would prefer another separate PR)